azure-pipelines.release-vnext-nightly.yml

```diff
@@ -80,6 +82,19 @@ jobs:
           artifactName: SBom-$(System.JobAttempt)
           targetPath: $(System.DefaultWorkingDirectory)/_manifest
 
+      # Since releases are scoped, this should warn for any packages that were mistakenly not included in scoping
+      - script: |
```

Comment:
If I'm not mistaken then this check will always fail just like how it fails on the current release. More importantly the version bump from nightly releases is never merged back into master. Therefore, it makes even less sense to include it here. For nightly releases we just care that eh released library works functionally

Reply:
Wouldn't this be a potential security issue? I assume all software we ship needs to be scrutinized, nightly or not.

Reply:
That's a good point, I'll edit this out